### PR TITLE
Test on laravel 5.5

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -24,9 +24,9 @@
         "mrclay/minify": "^2.2"
     },
     "require-dev": {
-        "graham-campbell/testbench": "^3.1",
+        "graham-campbell/testbench": "^3.1|^4.0",
         "mockery/mockery": "^0.9.4",
-        "phpunit/phpunit": "^4.8|^5.0"
+        "phpunit/phpunit": "^4.8|^5.0|^6.0"
```
Comment (GrahamCampbell) [inline composer.json]: Don't pull in phpunit 6. It won't work.

Comment (srmklive) [inline composer.json]: Tests are not working on in hhvm for Laravel 5.5.

Comment (srmklive) [inline composer.json]: Fixed

Comment (srmklive) [inline composer.json]: Although if i pull in phpunit 6, the tests only fail on hhvm for Laravel 5.5

Comment (GrahamCampbell) [reply composer.json]: > Do we need to support hhvm for 5.5?

I'd say no. :)

**Pull request merged**